test/distributed/org/apache/cassandra/distributed/impl/InstanceConfig.java

```diff
@@ -102,7 +107,7 @@ private InstanceConfig(int num,
                 .set("memtable_heap_space_in_mb", 10)
                 .set("commitlog_sync", "batch")
                 .set("storage_port", 7012)
-                .set("endpoint_snitch", SimpleSnitch.class.getName())
+                .set("endpoint_snitch", org.apache.cassandra.distributed.impl.Snitch.class.getName())
```

Comment:
Do we need a fully qualified name here?

Reply:
I went for the fully qualified name to try and clarify it wasn't a normal snitch - I think renaming the class is a better idea.